BUILD.md

```diff
@@ -67,6 +67,14 @@ LLVM 15 isn't supported due to compatibility issues with Boost dependency. The O
 API isn't support on MacOS, Vulkan must be used. Additionally Vulkan must be used through the 
 Molten-VK compatibility layer.  
 
+### Apple Silicon
+
+Cemu is an x86 (Intel) application and must be run through Rosetta. As such, you must also compile it under Rosetta. If you are running an ARM (Apple Silicon) based Mac, run the following command in your terminal before continuing.
+
+`arch -x86_64 zsh`
```

Comment:
`eval "$(/usr/local/Homebrew/bin/brew shellenv)"` needs to be run afterwards

Reply:
For simplicity, should I just add this to the "Installing brew" section? I don't think it'll cause any issues with Intel Macs. It's a little redundant though I suppose.

Reply:
I agree that `arch -x86_64 zsh` should just be added to the build steps.

Reply:
Sorry if there was confusion, in my comment I meant the `eval "$(/usr/local/Homebrew/bin/brew shellenv)"` command should be added to "Installing brew", not the arch command.